scripts/analysis/amwg_table.py

```diff
@@ -326,19 +355,33 @@ def _df_comp_table(write_html,output_location,case_names):
 
     df_comp['diff'] = df_comp['case'].values-df_comp['baseline'].values
 
+
     cols_comp = ['variable', 'unit', 'test', 'control', 'diff']
     df_comp.to_csv(output_csv_file_comp, header=cols_comp, index=False)
 
     #Create HTML output file name as well, if needed:
     if write_html:
         output_html_file_comp = output_location / "amwg_table_comp.html"
-
+    
     html = df_comp.to_html(index=False, border=1, justify='center', float_format='{:,.3g}'.format)  # should return string
-    # * * * * * * * * * * * * * * * * * * * * * * * * * * * *
-    #This will be for single-case for now,
-    #will need to think how to change as multi-case is introduced
-    # ie case_names[0] will need to be modified for more cases
-    preamble = f"""<html><head></head><body><h1>AMWG Case Comparison<h1><h2>Test Case: {case_names[0]}<br/>Control Case: {case_names[-1]}</h2>"""
+    preamble = f"""<html><head><title>ADF Mean Tables</title><link rel="stylesheet" href="../templates/adf_diag.css"></head><body >
+    
+    <nav role="navigation" class="primary-navigation">
+      <ul>
+        <li><a href="../index.html">Case Home</a></li>
+        <li><a href="../html_table/mean_table.html">Case Tables</a></li>
+        <li><a href="#">Links &dtrif;</a>
+          <ul class="dropdown">
+            <li><a href="https://www.cesm.ucar.edu">CESM</a></li>
+            <li><a href="https://www.cesm.ucar.edu/working_groups/Atmosphere/?ref=nav">AMWG</a></li>
+            <li><a href="https://www.cgd.ucar.edu/amp/">AMP</a></li>
+          </ul>
+        </li>
+        <li><a href="https://github.com/NCAR/ADF">About</a></li>
+        <li><a href="https://github.com/NCAR/ADF/discussions">Contact</a></li>
+      </ul>
+    </nav><h1>AMP Diagnostics</h1><h2>AMWG Case Comparison</h2><h2>Test Case: {case_names[0]}<br/>Control Case: {case_names[-1]}</h2>"""
```

Comment:
I might change this header to be something like `Model Diagnostics`, `CAM Diagnostics`, or `ADF Diagnostics`.